CHANGELOG.md

```diff
@@ -20,8 +20,9 @@ Root level:
 
 `location`:
 
-- [added] The `hint` field was added ([#102])
 - [added] The `country_code` field was added ([#108])
+- [added] The `areas` field was added ([#107])
+- [added] The `hint` field was added ([#102])
```

Comment:
The change in order seems unnecessary, but I don't care that much.

Reply:
I think that's an auto-merge thing by git, but I'm not sure.